Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -156,6 +156,32 @@ bool _CloneDWriteGlyphRun(_In_ DWRITE_GLYPH_RUN const* src, _Out_ DWRITE_GLYPH_R
 
             RETURN_NULL_IF_FAILED(textFormat->SetReadingDirection(dwriteDirection));
         }
+
+        CTLineBreakMode lineBreakMode;
+        if (CTParagraphStyleGetValueForSpecifier(settings,
+                                                 kCTParagraphStyleSpecifierLineBreakMode,
+                                                 sizeof(lineBreakMode),
+                                                 &lineBreakMode)) {
+            DWRITE_WORD_WRAPPING wrapping;
+            switch (lineBreakMode) {
+                case kCTLineBreakByClipping:
+                    wrapping = DWRITE_WORD_WRAPPING_NO_WRAP;
+                    break;
+                case kCTLineBreakByCharWrapping:
+                    wrapping = DWRITE_WORD_WRAPPING_CHARACTER;
+                    break;
+                // TODO 1121:: DWrite does not support line breaking by truncation, so just use default for now
+                case kCTLineBreakByTruncatingHead:
+                case kCTLineBreakByTruncatingTail:
+                case kCTLineBreakByTruncatingMiddle:
```

Comment:
Thats not a reasonable default for truncation.  LineBreakByClipping is more palatable instead.